fira-code-mode.el

```diff
@@ -107,7 +107,7 @@ will need to disable and re-enable the mode in order for the edits to take effec
 ;;;###autoload
 (define-minor-mode fira-code-mode
   "Fira Code ligatures minor mode"
-  :lighter  "  \xe15b"
+  :lighter " \xe15b"
```

Comment:
Normally I'd totally agree, but in this case when `fira-code-mode` is enabled, `\xe15b` is 2 characters wide, overlapping the character to its left. This makes it so that in order for it to look like it's preceded by one space, you need to put two spaces there - preceding by one space makes it look like there are none.
```suggestion
  :lighter "  \xe15b"
```